core/src/main/java/nl/nn/adapterframework/webcontrol/pipes/Webservices.java

```diff
@@ -207,16 +208,21 @@ private void zip(Adapter adapter, IPipeLineSession session)
 			NamingException {
 		Wsdl wsdl = new Wsdl(adapter.getPipeLine());
 		wsdl.setUseIncludes(true);
-		wsdl.setDocumentation(getDocumentation(wsdl, session));
+		wsdl.setDocumentation(getDocumentation(wsdl, adapter, session));
 		wsdl.init();
 		wsdl.zip(RestListenerUtils.retrieveServletOutputStream(session),
 				RestListenerUtils.retrieveSOAPRequestURL(session));
 	}
 
-	private String getDocumentation(Wsdl wsdl, IPipeLineSession session)
+	private String getDocumentation(Wsdl wsdl, Adapter adapter, IPipeLineSession session)
 			throws IOException {
-		return "Generated at " + RestListenerUtils.retrieveRequestURL(session)
+		String doc = "Generated at " + RestListenerUtils.retrieveRequestURL(session)
 				+ " as " + wsdl.getFilename() + getWsdlExtention() + " on "
 				+ DateUtils.getIsoTimeStamp() + ".";
+		String info = ((IXmlValidator)adapter.getPipeLine().getInputValidator()).getInfo();
+		if (info!=null) {
+			doc = doc + info;
+		}
+		return doc;
```

Comment:
Dit commentaar moet niet in de GUI er bij geplakt worden, maar onderdeel zijn van het genereren van de WSDL.
Ten eerste omdat dit niet in de GUI moet, en ten tweede omdat het ook in GUI 3.0 moet werken. (Daarom moet het ook niet in de GUI, zeg maar)